cpp/src/io/parquet/page_enc.cu

```diff
@@ -1088,8 +1088,8 @@ __global__ void __launch_bounds__(128) gpuDecideCompression(EncColumnChunk *chun
         if (comp_out[comp_idx].status != 0) { atomicAdd(&error_count, 1); }
       }
     }
-    uncompressed_data_size = WarpReduceSum32(uncompressed_data_size);
-    compressed_data_size   = WarpReduceSum32(compressed_data_size);
+    uncompressed_data_size = WarpReduceSum(uncompressed_data_size);
+    compressed_data_size   = WarpReduceSum(compressed_data_size);
```

Comment:
This WarpReduceSum is called inside an if branch `if (t < 32)` where `t` is `threadIdx.x`. @vuule Does this sound right? Should this not be an issue?

Reply:
I can't see the issue, first 32 threads will call this and they make up the warp that will contribute to the reduction. What am I missing?
cc @devavret as he's way more familiar with this code.